VKClone/ViewControllers/MainViewController.swift

```diff
@@ -0,0 +1,72 @@
+// MainViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
```

Comment:
```suggestion
import UIKit

```
Поле импорта всегда оставляй отступ пожалуйста не забывай